protocol/src/main/java/com/amazon/opendistroforelasticsearch/sql/protocol/response/format/CsvResponseFormatter.java

```diff
@@ -26,119 +26,14 @@
-      return String.join(INTERLINE_SEPARATOR, headersAndData);
-    }
-
-    private String getHeaderLine(QueryResult response, boolean sanitize) {
-      List<String> headers = getHeaders(response, sanitize);
-      return String.join(INLINE_SEPARATOR, headers);
-    }
-
-    private List<String> getDataLines(QueryResult response, boolean sanitize) {
-      List<List<String>> data = getData(response, sanitize);
-      return data.stream().map(v -> String.join(INLINE_SEPARATOR, v)).collect(Collectors.toList());
-    }
-
-    private List<String> getHeaders(QueryResult response, boolean sanitize) {
-      ImmutableList.Builder<String> headers = ImmutableList.builder();
-      response.columnNameTypes().forEach((column, type) -> headers.add(column));
-      List<String> result = headers.build();
-      return sanitize ? sanitizeHeaders(result) : result;
-    }
-
-    private List<List<String>> getData(QueryResult response, boolean sanitize) {
-      ImmutableList.Builder<List<String>> dataLines = new ImmutableList.Builder<>();
-      response.iterator().forEachRemaining(row -> {
-        ImmutableList.Builder<String> line = new ImmutableList.Builder<>();
-        // replace null values with empty string
-        Arrays.asList(row).forEach(val -> line.add(val == null ? "" : val.toString()));
-        dataLines.add(line.build());
-      });
-      List<List<String>> result = dataLines.build();
-      return sanitize ? sanitizeData(result) : result;
-    }
-
-    /**
-     * Sanitize CSV headers because Elasticsearch allows special character present in field names.
-     */
-    private List<String> sanitizeHeaders(List<String> headers) {
-      return headers.stream()
-          .map(this::sanitizeCell)
-          .map(cell -> quoteIfRequired(INLINE_SEPARATOR, cell))
-          .collect(Collectors.toList());
-    }
-
-    /**
-     * Sanitize CSV lines in which each cell is sanitized to avoid CSV injection.
-     */
-    private List<List<String>> sanitizeData(List<List<String>> lines) {
-      List<List<String>> result = new ArrayList<>();
-      for (List<String> line : lines) {
-        result.add(line.stream()
-            .map(this::sanitizeCell)
-            .map(cell -> quoteIfRequired(INLINE_SEPARATOR, cell))
-            .collect(Collectors.toList()));
-      }
-      return result;
-    }
-
-    private String sanitizeCell(String cell) {
-      if (isStartWithSensitiveChar(cell)) {
-        return "'" + cell;
-      }
-      return cell;
-    }
-
-    private String quoteIfRequired(String separator, String cell) {
-      final String quote = "\"";
-      return cell.contains(separator)
-          ? quote + cell.replaceAll("\"", "\"\"") + quote : cell;
-    }
-
-    private boolean isStartWithSensitiveChar(String cell) {
-      return SENSITIVE_CHAR.stream().anyMatch(cell::startsWith);
-    }
+  public CsvResponseFormatter(boolean sanitize) {
+    super(",", sanitize);
+    System.out.println("Harold sanitize : " + sanitize);
```

Comment:
Could you remove all the sysout? There is another one at line 44 in `FlatResponseFormatter`

Reply:
Sorry, removed.

Reply:
Thanks! 👍